extensions/spark/kyuubi-spark-connector-hive/src/main/scala/org/apache/kyuubi/spark/connector/hive/read/HiveScanBuilder.scala

```diff
@@ -37,6 +37,8 @@ case class HiveScanBuilder(
       catalogTable = table,
       dataSchema = dataSchema,
       readDataSchema = readDataSchema(),
-      readPartitionSchema = readPartitionSchema())
+      readPartitionSchema = readPartitionSchema(),
+      partitionFilters = partitionFilters,
+      dataFilters = dataFilters)
```

Comment:
Does `dataFilters` affects correctness?

Reply:
It is designed as a kind of file pruning, so it only affect performance. For this hive datasource v2, I think it has no effect.

Reply:
Yes, `dataFilters` has no effect on correctness. 

KSHC does not support pushdown filters to file sources, so, in fact, both `dataFilters` and `pushedFilters` are useless.